pkg/kvstore/etcd.go

```diff
@@ -211,6 +211,13 @@ func (e *etcdModule) newClient(ctx context.Context, opts *ExtraOptions) (Backend
 func init() {
 	// register etcd module for use
 	registerBackend(EtcdBackendName, etcdInstance)
+
+	if duration := os.Getenv("CILIUM_ETCD_STATUS_CHECK_INTERVAL"); duration != "" {
```

Comment:
@tgraf any reason not to have this as a CLI option as well?

Reply:
Yes. This option is very unlikely to be changed frequently and a full-blown CLI option makes backporting harder and also pollutes the `--help` output massively. We already have way too many options.

The environment variable is the last resort for advanced users

Reply:
Got it, makes sense.

```
We already have way too many options.
``` 

+1 to the above.